beacon-chain/state/v1/field_roots.go

```diff
@@ -170,46 +171,54 @@ func (h *stateRootHasher) computeFieldRootsWithHasher(ctx context.Context, state
 	fieldRoots[13] = randaoRootsRoot[:]
 
 	// Slashings array root.
-	slashingsRootsRoot, err := ssz.SlashingsRoot(state.Slashings)
+	slashingsRootsRoot, err := ssz.SlashingsRoot(state.Slashings())
 	if err != nil {
 		return nil, errors.Wrap(err, "could not compute slashings merkleization")
 	}
 	fieldRoots[14] = slashingsRootsRoot[:]
 
 	// PreviousEpochAttestations slice root.
-	prevAttsRoot, err := h.epochAttestationsRoot(state.PreviousEpochAttestations)
+	prevAtts, err := state.PreviousEpochAttestations()
```

Comment:
These changes other than genesis time seem unrelated to the PR but I get what we are doing here. Should be ok to squeeze it

Reply:
I had to use `state.GenesisTime()` because I removed the field from the proto state. But the method is present in the interface, so I had to change the param type as well. This in turn made all fields calls invalid, so I had to use methods everywhere. Butterfly effect.